en/config_heli/README.md

```diff
@@ -0,0 +1,87 @@
+# Helicopter Configuration
+
+This section contains topics related to helicopter configuration and tuning.
+
+:::note
+Helicopters are less well-supported than other types of vehicles.
+We would welcome your [contribution](../contribute/README.md) of new features, new frame configurations, or other improvements.
+:::
+
+## Supported Configurations
+
+Supported helicopter configurations:
+- Single main rotor with any type of swash-plate
+- ESC tail rotor
+
+
+## Setup
+
+To setup and configure a helicopter: 
+
+1. Select the helicopter [Airframe](../config/airframe.md) in QGroundControl
+1. Configure [Actuator](../config/actuators.md) geometry
+   ![Geometry helicopter](../../assets/config/actuators/qgc_geometry_helicopter.png)
+   1. Remove the rotor blades and propellers
+   1. Configure the swash-plate servos.
+      The angle is measured clockwise with 0 pointing forwards.
+      Trim is typically not needed, but can be used to offset individual servo positions.
+   1. Set the main rotor turning orientation
+1. Assign motors and servos to outputs and test (also in [Actuator configuration](../config/actuators.md))
+   1. Assign the motors and servos to the outputs
+   1. Power the vehicle with a battery and use the actuator testing sliders to validate correct servo and motor assignment and direction.
+1. Using an RC in [Acro mode](../flight_modes/acro_mc.md), verify the correct movement of the swash-plate:
+   - moving the Roll stick to the right should tilt the swash-plate to the right.
+   - moving the Pitch stick forward should tilt the swash-plate forward.
+1. Arm the vehicle and check the main rotor spins up slowly.
+   Adjust the throttle spoolup time as needed.
+   You can also adjust the throttle curve with the parameters [CA_HELI_THR_Cx](../advanced_config/parameter_reference.md#CA_HELI_THR_C0).
+   The default is constant, maximum throttle.
```

Comment:
I presume that the throttle curve adjusts the throttle non-linearly across the RC stick position for some reason. Is there something we can/should link here to make this clear, or to advise what makes sense for most users?

Reply:
Yes. The default makes sense for most. I don't have enough experience here either.